package.json

```diff
@@ -14,15 +14,16 @@
     "migrate:dev": "sequelize db:migrate",
     "migrate:test": "cross-env NODE_ENV=test sequelize db:migrate",
     "migrate:prod": "cross-env NODE_ENV=production sequelize db:migrate",
-    "undo:dev": "sequelize db:migrate:undo",
-    "undo:test": "cross-env NODE_ENV=test sequelize db:migrate:undo",
+    "undo:dev": "sequelize db:migrate:undo:all",
+    "undo:test": "cross-env NODE_ENV=test sequelize db:migrate:undo:all",
     "undo:prod": "cross-env NODE_ENV=prooduction sequelize db:migrate:undo",
     "seed:dev": "sequelize db:seed:all",
     "seed:test": "cross-env NODE_ENV=test sequelize db:seed:all"
   },
   "author": "Andela Simulations Programme",
   "license": "MIT",
   "dependencies": {
+    "@hapi/joi": "^15.1.1",
```

Comment:
What is the use of this package. can you provide resource on the slack channel ?

Reply:
@hapi/joi is an npm package used for validation.